Ogre Color not defined

With new ros melodic packages from today (April 23rd 2020) the packages does not build anymore due to the missing definition of `Ogre::ColourValue`

```bash
Errors   /VectorAtPositionDisplay.hpp:100:21: error: field ‘color_’ has incomplete type ‘Ogre::ColourValue’
   Ogre::ColourValue color_;
```

It looks like a missing include somewhere, which was previously  included by some rviz header, but not anymore.

--- a/kindr_rviz_plugins/include/kindr_rviz_plugins/VectorAtPositionDisplay.hpp
+++ b/kindr_rviz_plugins/include/kindr_rviz_plugins/VectorAtPositionDisplay.hpp
@@ -9,6 +9,9 @@
 
 // kindr ros
 #include <kindr_msgs/VectorAtPosition.h>
+
+// ogre
+#include <OGRE/OgreColourValue.h>
 #endif
 
 

